roles/ood_igv/files/script.sh.erb

```diff
@@ -28,10 +28,19 @@ cd "${HOME}"
 # Start Integrative Genomics Viewer
 #
 
+# Determine memory for JVM in IGV
+default_jvm_other_mb=512 # default to 0.5 GB
+total_available_mb=$(($SLURM_MEM_PER_CPU * $SLURM_CPUS_PER_NODE))
```

Comment:
I don't see these two variables set in a job.

Reply:
didn't know this only set in a sbatch job, which is the way we start IGV job.
So I see it now.